node/api.go

```diff
@@ -69,6 +69,7 @@ func (api *PublicBlockChainAPI) GetNodeInfo() (interface{}, error) {
 			CuckatooDiff: getDifficultyRatio(cuckatooNodes, api.node.node.Params, pow.CUCKATOO),
 		},
 		TestNet:          api.node.node.Config.TestNet,
+		MixNet:           api.node.node.Config.MixNet,
```

Comment:
the field should be a network type `{mainnet, testnet, mixnet}` for the current running network, instead of a boolean type for every network.